archlinuxcn/keypunch-git/PKGBUILD

```
@@ -0,0 +1,39 @@
+#Maintainer:    mumi jim <echo "=02bj5yav9Gb0V3bA1Waq9VatVXb" | rev | base64 -d>
+# Contributor: Klaus Alexander Seiﬆrup <$(echo 0x1fd+d59decfa=40 | tr 0-9+a-f=x ka-i@p-u.l)>
+
+#Thanks Klaus Alexander Seistrup :)
+#this PKGBUILD of Keypunch
+#github Repo: https://github.com/bragefuglseth/keypunch/
+
+pkgname="keypunch-git"
+_appname="keypunch"
+pkgver=6.2.r8.g77f9956-1
+pkgrel=1
+pkgdesc='Practice your typing skills'
+url='https://github.com/bragefuglseth/keypunch'
+_app_website='https://apps.gnome.org/Keypunch'
+arch=('aarch64' 'x86_64')
+license=('GPL-3.0-or-later')
+source=("git+$url.git")
+sha512sums=('SKIP')
+provides=('keypunch')
+conflicts=('keypunch')
+depends=('gtk4' 'libadwaita')
+makedepends=('blueprint-compiler' 'gettext' 'git' 'meson' 'ninja' 'pkgconf' 'rust')
```

```suggestion
makedepends=('blueprint-compiler' 'git' 'meson' 'ninja' 'cargo')
```

- According to [Arch Wiki/PKGBUILD](https://wiki.archlinux.org/title/PKGBUILD#makedepends):
> The package [base-devel](https://archlinux.org/packages/?name=base-devel) is assumed to be already installed when building with makepkg. Dependencies of this package **should not** be included in makedepends array.

- According to [Rust package guidelines](https://wiki.archlinux.org/title/Rust_package_guidelines#Depends):
> As such, the tool most PKGBUILDs are going to call is cargo and you should depend directly on it.